lib/sidekiq_logger_formatter.rb

```diff
@@ -0,0 +1,29 @@
+class SidekiqLoggerFormatter < Logger::Formatter
+  # This method is stdlib in ruby :reek:LongParameterList { max_params: 4 }
+  def call(severity, time, progname, msg)
+    msg = filter_msg(msg)
+    super(severity, time, progname, msg)
+  end
+
+  private
+
+  def filter_msg(msg)
+    filter_msg_string(msg) if msg.is_a? String
```

Comment:
When `msg` is a String, this method is supposed to return a filtered copy of `msg` and not mutate it in place right? I think this will not work in that case....since it will just fall through to the next line and return `nil`?

I think it needs to look like this?

```ruby
def filter_msg(msg)
  if msg.is_a?(String)
    filter_msg_string(msg)
  elsif msg.is_a?(Hash)
    filter_msg_hash(msg)
  end
end
```